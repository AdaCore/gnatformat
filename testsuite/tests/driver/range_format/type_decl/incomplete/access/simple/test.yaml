description: ConcreteTypeDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 59",
    "-EL 60",
    "--pipe",
  ]
