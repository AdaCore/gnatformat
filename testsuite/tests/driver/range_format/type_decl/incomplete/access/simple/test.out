test.ads(<ConcreteTypeDecl ["Link"] test.ads:59:4-59:29>) - 59:1-59:29
^
   type Link is access Cell;$

